package.json

```diff
@@ -28,7 +28,7 @@
     "generate": "ts-node ./scripts/generate",
     "build": "argo-run build",
     "server": "yarn start",
-    "start": "argo-run serve --port 39351",
+    "start": "argo-run serve --",
```

Comment:
We should remove ` --` too. Shopify App Cli automatically add ` -- ` to command argument. Doing `yarn start --port=12345` with `start: argo-run serve --` doesn't work for yarn.